src/main/java/com/snowflake/kafka/connector/internal/streaming/telemetry/SnowflakeTelemetryChannelStatus.java

```diff
@@ -75,6 +87,28 @@ public SnowflakeTelemetryChannelStatus(
     }
   }
 
+  @Override
+  public boolean isEmpty() {
```

Comment:
I assume this is used to not send data when everything is default? If yes, I think we should always send since it can help us to know a restart or a table is dropped so everything is reset, wdyt?

Reply:
Correct that is what it is used for. Makes sense to send this regardless of state since this is the 'close' event corresponding with the reportKafkaPartitionStart event. 

Made the change in SnowflakeTelemetryService.java, this means we're making a behavior change to mirror this behavior into Snowpipe